app/src/main/java/com/example/henzoshimada/feeltrip/Mood.java

```diff
@@ -219,9 +263,11 @@ public Date getDate() {
     }
 
     public void setDate(Date date) {
-        this.date = date.getTime();
-        this.trueDate = date;
-        setStateByIndex(2);
+        if (this.date != date.getTime()) {
+            this.date = date.getTime();
+            this.trueDate = date;
+            setStateByIndex(2);
```

Comment:
but the user should be able to pick the date for every post instead of being system time